contrib/libvpl/module.defs

```diff
@@ -1,11 +1,11 @@
 $(eval $(call import.MODULE.defs,LIBVPL,libvpl))
 $(eval $(call import.CONTRIB.defs,LIBVPL))
 
-LIBVPL.FETCH.url       = https://github.com/oneapi-src/oneVPL/archive/refs/tags/v2022.1.2.tar.gz
-LIBVPL.FETCH.url      += https://github.com/HandBrake/HandBrake-contribs/releases/download/contribs/oneVPL-2022.1.2.tar.gz
-LIBVPL.FETCH.sha256    = 8ff93be017273eefa83dc6cd7f5d82facb168e38eafabafa2e9613a4e742016f
-LIBVPL.FETCH.basename  = oneVPL-2022.1.2.tar.gz
-LIBVPL.EXTRACT.tarbase = oneVPL-2022.1.2
+LIBVPL.FETCH.url       = https://github.com/oneapi-src/oneVPL/archive/refs/tags/v2023.1.0.tar.gz
+LIBVPL.FETCH.url      += https://github.com/oneapi-src/oneVPL/archive/refs/tags/v2023.1.0.tar.gz
```

Comment:
Please add oneVPL dispatcher into HandBrake-contribs. I will update the link

Reply:
Done.

Reply:
Thanks.